database/migrations/2025_02_10_161202_create_articles_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('articles', function (Blueprint $table) {
+            $table->id()->autoIncrement();
+            $table->timestamps();
+            $table->string('title');
+            $table->string('author');
+            $table->string('short_description');
+            $table->string('text');
```

Comment:
String size is limited. To prevent that, use the `$table->text()` attribute creator